msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -10,55 +10,62 @@ ms.date: 04/07/2022
 
 # Apps for Teams meetings and calls
```

Comment:
@v-sdhakshina  I don't see a reference of Meeting apps capabilities in https://support.microsoft.com/en-us/office/meetings-1f8862a1-48f2-4869-863d-564123343d38#ID0EBBD=Participate; Should we add one or more sections?! 
cc: Manasi Sharma

Reply:
do it as separate task